llama_index/llms/konko_utils.py

```diff
@@ -219,12 +236,27 @@ async def acompletion_with_retry(
     is_chat_model: bool, max_retries: int, **kwargs: Any
 ) -> Any:
     """Use tenacity to retry the async completion call."""
+    try:
+        import konko
+
+    except ImportError:
+        raise ValueError(
+            "Could not import konko python package. "
+            "Please install it with `pip install konko`."
+        )
```

Comment:
this is quite repetitive. we should extract a function and reuse.

Reply:
done